src/pages/capacities/CapacitiesTable.tsx

```diff
@@ -129,7 +132,7 @@ export const CapacitiesTable: React.FC<CapacitiesTableProps> = ({
             <Tooltip trigger={<InfoOutlineIcon />}>
               <Text color="grey600" weight={600} size={12}>
                 Duration capacity commitment in epochs. Currently, one epoch is
-                set as 24 hours.
+                set as {formatDistanceToNowStrict(Date.now() + epochDuration)}
```

Comment:
I guess you can use just `formatDistanceStrict` without `Date.now()`

Reply:
I added `formatDuration` fn